ui/raidboss/data/07-dt/raid/r7s.ts

```diff
@@ -1,14 +1,421 @@
+    },
+    {
+      id: 'R7S Brutish Swing',
+      type: 'StartsUsing',
+      netRegex: {
+        id: ['A592', 'A593', 'A5A3', 'A5A5'],
+        source: 'Brute Abombinator',
+        capture: true,
+      },
+      durationSeconds: (_data, matches) => parseFloat(matches.castTime) + 10,
+      alertText: (data, matches, output) => {
+        const id = matches.id;
+        switch (id) {
+          case 'A592':
+            return output.out!();
+          case 'A593':
+            return output.in!();
+          case 'A5A3': {
+            if (data.brutalImpactCount > 7)
+              return output.out!();
+
+            const followup = data.stoneringer2Followup ? output.bigAoe!() : output.awayFromFront!();
+            return output.outFollowup!({ followup: followup });
+          }
+          case 'A5A5': {
+            if (data.brutalImpactCount > 7)
+              return output.in!();
+
+            const followup = data.stoneringer2Followup ? output.bigAoe!() : output.awayFromFront!();
+            return output.inFollowup!({ followup: followup });
+          }
+        }
+      },
+      run: (data) => {
+        delete data.storedStoneringer;
+        delete data.stoneringer2Followup;
+      },
+      outputStrings: {
+        in: {
+          en: 'In at tethered wall',
+        },
+        out: {
+          en: 'Out from tethered wall',
+        },
+        inFollowup: {
+          en: 'In at tethered wall + ${followup}',
+        },
+        outFollowup: {
+          en: 'Out from tethered wall + ${followup}',
+        },
+        awayFromFront: {
+          en: 'Spread, Away from front',
+        },
+        bigAoe: Outputs.bigAoe,
+      },
+    },
+    {
+      id: 'R7S Glower Power',
+      type: 'StartsUsing',
+      netRegex: { id: 'A94C', source: 'Brute Abombinator', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Spread, Away from front',
+        },
+      },
+    },
+    {
+      id: 'R7S Revenge of the Vines',
+      type: 'StartsUsing',
+      netRegex: { id: 'A587', source: 'Brute Abombinator', capture: false },
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'R7S Thorny Deathmatch',
+      type: 'Tether',
+      netRegex: { id: '0152', capture: true },
+      infoText: (_data, matches, output) => output.text!({ target: matches.target }),
+      outputStrings: {
+        text: {
+          en: 'Tank tether on ${target}',
+        },
+      },
+    },
+    {
+      id: 'R7S Abominable Blink',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.flareMarker, capture: true },
+      alertText: (data, matches, output) => {
+        if (matches.target === data.me)
+          return output.flare!();
+        return output.avoidFlare!();
+      },
+      outputStrings: {
+        avoidFlare: {
+          en: 'Away from Flare',
+        },
+        flare: {
+          en: 'Flare on YOU, Away from party',
```

Comment:
```suggestion
          en: 'Flare + buster on YOU',
```
No strong feelings on this, but in theory someone who's in Savage should be aware of how to handle a Flare, while it's definitely not intuitive that this iteration is a buster on the target.